### PR TITLE
ALEC-94: Update health check command

### PR DESCRIPTION
**Pull request by emarsh1995**
- Update `admin/distributed_install.adoc`
- Update `install/basic_install.adoc`, `/pre_install.adoc`
- Update `reference/datasources/kafka.adoc`

## External References

- Jira (Issue Tracker): https://opennms.atlassian.net/browse/ALEC-94

### CODE REVIEW

**Review by agalue — DISMISSED** [review]
It's been years since I last used ALEC, but the changes look good to me from what I remember.

File: docs/modules/admin/pages/distributed_install.adoc [hunk 1]
```diff
@@ -21,110 +23,97 @@ In addition to an existing instance of OpenNMS Horizon 30.0.1 or newer, these in
 
 == OpenNMS configuration
 
-Complete the following steps in the OpenNMS Karaf shell.
-They configure your OpenNMS instance in preparation for a distributed installation of ALEC.
+Follow these steps to configure your OpenNMS instance for a distributed installation of ALEC.
+The steps should be completed using the OpenNMS Karaf shell.
```
Comment (Bonrob2) [inline docs/modules/admin/pages/distributed_install.adoc]:
```suggestion
You should use the OpenNMS Karaf shell to complete these steps. 
```

File: docs/modules/admin/pages/distributed_install.adoc [hunk 2]
```diff
@@ -21,110 +23,97 @@ In addition to an existing instance of OpenNMS Horizon 30.0.1 or newer, these in
...
++
+[source, shell]
 yum -y install opennms-alec-plugin
-----
-
-.Configure the feature set to be installed when OpenNMS starts:
 
-[source, karaf]
-----
+. Configure the feature set to be installed when OpenNMS starts:
```
Comment (Bonrob2) [inline docs/modules/admin/pages/distributed_install.adoc]:
```suggestion
. Configure the feature set to install when OpenNMS starts:
```

File: docs/modules/admin/pages/distributed_install.adoc [hunk 3]
```diff
@@ -21,110 +23,97 @@ In addition to an existing instance of OpenNMS Horizon 30.0.1 or newer, these in
...
+. Configure ZooKeeper for distributed coordination:
++
+[source, shell]
 echo 'connectString=127.0.0.1:2181' > "$SENTINEL_HOME/etc/org.opennms.features.distributed.coordination.zookeeper.cfg"
-----
-
-.Configure the feature set to be installed when Sentinel starts:
 
-[source, karaf]
+. Configure the feature set to be installed when Sentinel starts:
```
Comment (Bonrob2) [inline docs/modules/admin/pages/distributed_install.adoc]:
```suggestion
. Configure the feature set to install when Sentinel starts:
```

File: docs/modules/install/pages/pre_install.adoc [hunk 4]
```diff
@@ -2,28 +2,29 @@
...
+* *RHEL/CentOS:* `export OPENNMS_HOME=/opt/opennms`
+* *Ubuntu/Debian:* `export OPENNMS_HOME=/usr/share/opennms`
 
-=== Verify your OpenNMS installation
+=== Verify OpenNMS installation
 
-Run the `health:check` command in an OpenNMS Karaf shell to verify the status of your OpenNMS instance.
+Run `opennms:health-check` in an OpenNMS Karaf shell to verify the status of your Horizon or Meridian instance.
+If the command returns `Everything is awesome`, you can continue to <<alarm-history-storage, Enable alarm history storage>>.
+If it returns `Oh no, something is wrong`, you must address the underlying problem before moving forward.
```
Comment (Bonrob2) [inline docs/modules/install/pages/pre_install.adoc]:
```suggestion
If it returns `Oh no, something is wrong`, you must address the underlying problem before continuing.
```

File: docs/modules/install/pages/pre_install.adoc [hunk 5]
```diff
@@ -34,92 +35,88 @@ Connecting to ElasticSearch ReST API (Flows)   [ Success ] => Not configured
...
-[source]
+[source, xml]
 ----
 <service>
     <name>OpenNMS:Name=Syslogd</name>
 ----
 
-This change will come into effect the next time OpenNMS restarts.
-Do not restart OpenNMS at this point; you will restart it as part of the ALEC installation procedure.
+This change is effected the next time OpenNMS restarts.
```
Comment (Bonrob2) [inline docs/modules/install/pages/pre_install.adoc]:
```suggestion
This change occurs the next time OpenNMS restarts.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
minor changes, thank you!

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Pull request merged**